osu.Game.Tournament/Components/TournamentBeatmapPanel.cs

```diff
@@ -192,5 +185,86 @@ private void updateState()
                 Alpha = 1;
             }
         }
+
+        private class ModSprite : Container
```

Comment:
should probably be:
 1. moved to a separate file.
 2. renamed to `TournamentModDisplay` or something of the sorts.
 3. inherit `CompositeDrawable` instead of `Container`, shouldn't be able to mutate children from outside that component.